tests/codeception/acceptance/Media/MediaCest.php

```diff
@@ -142,6 +142,27 @@ public function testAllowedEmbedCodes(AcceptanceTester $I) {
       $I->click('Save');
       $I->canSee('has been created');
     }
+
+  }
+
+  /**
+   * Test allowed tags for administrators
+   *
+   * @group embed-codes
+   */
+  public function testAllowedTags(AcceptanceTester $I) {
+    $I->logInWithRole('administrator');
+    $I->amOnPage('/media/add/embeddable');
+    $embed_name = $this->faker->words(3, TRUE);
+    $I->fillField('Name', $embed_name);
+    $I->fillField('Embed Code', '<a></a> <aside></aside> <audio></audio> <b></b> <blockquote></blockquote> <br> <button></button> <caption></caption> <cite></cite> <code></code> <dd></dd> <dl></dl> <div></div> <drupal-media></drupal-media> <dt></dt> <em></em> <embed></embed> <form></form> <h1></h1> <h2></h2> <h3></h3> <h4></h4> <h5></h5> <hr> <i></i> <img /> <input /> <label></label> <li></li> <object></object> <ol></ol> <option></option> <p></p> <param /> <pre></pre> <s></s> <script></script> <select></select> <source></source> <span></span> <strong></strong> <sub></sub> <sup></sup> <table></table> <tbody></tbody> <textarea></textarea> <td></td> <tfoot></tfoot> <th></th> <thead></thead> <tr></tr> <ul></ul> <video></video>');
+    $I->click('Save');
+    $I->seeCurrentUrlEquals('/admin/content/media');
+    $I->canSee('has been created');
+    $I->canSee($embed_name);
+    $I->click($embed_name);
+    $I->canSee('<a></a> <aside></aside> <audio></audio> <b></b> <blockquote></blockquote> <br> <button></button> <caption></caption> <cite></cite> <code></code> <dd></dd> <dl></dl> <div></div> <drupal-media></drupal-media> <dt></dt> <em></em> <embed></embed> <form></form> <h1></h1> <h2></h2> <h3></h3> <h4></h4> <h5></h5> <hr> <i></i> <img /> <input /> <label></label> <li></li> <object></object> <ol></ol> <option></option> <p></p> <param /> <pre></pre> <s></s> <script></script> <select></select> <source></source> <span></span> <strong></strong> <sub></sub> <sup></sup> <table></table> <tbody></tbody> <textarea></textarea> <td></td> <tfoot></tfoot> <th></th> <thead></thead> <tr></tr> <ul></ul> <video></video>');
```

Comment:
I don't believe this test actually checks for the change because the `allowed_tags` are implemented upon rendering, not in the form.
I think we can remove the test completely.

Reply:
can do.

Reply:
test removed.